cmd/lakectl/cmd/fs_upload.go

```diff
@@ -32,13 +32,16 @@ var fsUploadCmd = &cobra.Command{
 
 		ctx, stop := signal.NotifyContext(ctx, os.Interrupt, os.Kill)
 		defer stop()
-
-		stat, err := os.Stat(source)
-		if err != nil {
-			Die("failed to stat source", 1)
+		ignoreRecessive := true // Ignore recursive if source is the standard input or a file (not a directory)
```

Comment:
"recessive"? 🧬 
Possibly spelling correction gone wrong.

Reply:
Too much trust on the auto-correct 😞 
Thanks

Reply:
Still recessive 🙀 .